client/config/config.go

```diff
@@ -92,220 +48,104 @@ func Cmd(defaultCLIHome string) *cobra.Command {
 
 	cmd.Flags().String(flags.FlagHome, defaultCLIHome,
 		"set client's home directory for configuration")
-	cmd.Flags().Bool(flagGet, false,
-		"print configuration value or its default if unset")
+	//	cmd.Flags().Bool(flagGet, false,
```

Comment:
I think we dont need a flagGet flag here, do we?

Reply:
that's correct! as per Alessio's proposal https://github.com/cosmos/cosmos-sdk/pull/8820#discussion_r590599253